tests/integration/test_studio_editable.py

```diff
@@ -0,0 +1,299 @@
+from xblock.core import XBlock
+from xblock.fields import Boolean, Dict, Float, Integer, List, String
+from xblock.validation import ValidationMessage
+from xblockutils.studio_editable import StudioEditableXBlockMixin
+from xblockutils.studio_editable_test import StudioEditableBaseTest
+
+
+class EditableXBlock(StudioEditableXBlockMixin, XBlock):
+    """
+    A Studio-editable XBlock
+    """
+    color = String(default="red")
+    count = Integer(default=42)
+    comment = String(default="")
+    editable_fields = ('color', 'count', 'comment')
+
+    def validate_field_data(self, validation, data):
+        """ Basic validation method for these tests """
+        if data.count <=0:
```

Comment:
Are you guys using any linters on this repo? This is a pep8 violation (space between operator & number)